types/vscode/index.d.ts

```diff
@@ -1480,7 +1474,7 @@ declare module 'vscode' {
          * the `skipEncoding`-argument: `uri.toString(true)`.
          *
          * @param skipEncoding Do not percentage-encode the result, defaults to `false`. Note that
-         *    the `#` and `?` characters occurring in the path will always be encoded.
+         *	the `#` and `?` characters occurring in the path will always be encoded.
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/vscode/index.d.ts;
  Line: 1477;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.